db/migrations/202410141503_wallet_child_idx.go

```diff
@@ -0,0 +1,26 @@
+package migrations
```

Comment:
is this still needed? or can we use the app ID as the derivation path?

Reply:
refactored to use wallet_child_pubkey